.github/workflows/assign-to-support-project.yml

```diff
@@ -0,0 +1,41 @@
+name: Assign to Support Project board
+# Assigns new issues to project based on label
+# See https://github.com/orgs/department-of-veterans-affairs/projects/1408/
+# Organizes GitHub issue into respective column based on label
+
+on:
+  issues:
+    types: [opened]
```

Comment:
This should suffice 👍 
If an issue is opened without one of these labels and then one of the labels is added later, this action won't run, since it only opens on newly-opened issues. But I think that's okay because 'T1-access-request', 'new-vfs-team-member' and 'new-vfs-team' are added automatically when those specific issues are created.

Reply:
Thanks Rebecca! I added that to this workflow